rust/gui-client/src-tauri/src/client/debug_commands.rs

```diff
@@ -9,20 +9,16 @@ pub enum Cmd {
     CheckForUpdates,
     Crash,
     DnsChanges,
-    Hostname,
     NetworkChanges,
 }
 
 pub fn run(cmd: Cmd) -> Result<()> {
     match cmd {
-        Cmd::CheckForUpdates => check_for_updates()?,
-        Cmd::Crash => crash()?,
-        Cmd::DnsChanges => client::network_changes::run_dns_debug()?,
-        Cmd::Hostname => hostname(),
```

Comment:
I haven't used the `debug hostname` subcommand in months, so I got rid of it.

Reply:
SGTM. I'd prefer these refactorings to go in separate PRs in case we need to revert something.

Reply:
It's in #5363 now